packages/libs/deer/src/error/mod.rs

```diff
@@ -83,6 +104,78 @@ impl Id {
     }
 }
 
+// TODO: most likely (in 0.2) we want to actually have a proper schema
+pub struct Schema {
+    ty: String,
```

Comment:
I wonder, if we want `String` or `Cow<str>` to support non-owned values, but probably it's not worth the hassle as we have a big struct next to it anyway.

Reply:
I also thought of that, but because `other` is already a full-blown `BTreeMap`, and you likely define more than just the type, it wasn't worth the effort (for now).